evap/staff/tools.py

```diff
@@ -68,6 +68,13 @@ def delete_navbar_cache_for_users(users):
         cache.delete(key)
 
 
+def create_user_list_html_string_for_message(users):
+    msg = format_html("")
+    for user in users:
+        msg += format_html("<br />{} {} ({})", user.first_name, user.last_name, user.email)
+    return msg
```

Comment:
There exists a [`format_html_join`](https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.html.format_html_join) which could make this more compact

Reply:
Cool, didn't see that. I think this is the only occurence where it really makes sense to prefer this over `format_html` -- do you agree?

Reply:
Yes, I think that’ll be the only occurrence